debian/rules

```diff
@@ -51,7 +49,7 @@ override_dh_shlibdeps:
 override_dh_virtualenv:
 	./debian/build_virtualenv
 
-# We are restricted to compat level 9 (because xenial), so have to
-# enable the systemd bits manually.
+# TODO: Experiment with --buildsystem=dh_virtualenv
+# https://dh-virtualenv.readthedocs.io/en/latest/usage.html#experimental-buildsystem-support
 %:
-	dh $@ --with python-virtualenv --with systemd
+	dh $@ --with python-virtualenv --buildsystem=pybuild
```

Comment:
note to my future self: previously we used the `python_distutils` build system; switching to `pybuild` matches the recommendation at https://wiki.debian.org/Python/LibraryStyleGuide. I don't think there was a good reason why we used `python_distutils`, other than that it was the default.